source/enumerate-databases.rst

```diff
@@ -2,14 +2,8 @@
 Enumerating Databases
 =====================
 
-:Spec Title: Enumerating Databases
-:Spec Ticket: SPEC-865
-:Spec Version: 1.0
-:Author: Jeremy Mikola
 :Status: Accepted
-:Type: Standards
 :Minimum Server Version: 3.8
```

Comment:
I don't recall why this server version was used, but I wasn't confident enough to change it. The most recent git blame on the file had it bumped from 3.7 to 3.8 in https://github.com/mongodb/specifications/commit/3d13d39aea4eeed590989cf6fd649d09c0b72cd2#diff-be10ea70ab383a03cf3f821429f4782517fcbbef5a82b9ff43a9e44a04fd3e79R11. It was originally 3.6 dating back to the original spec version, before being bumped to 3.7 by https://github.com/mongodb/specifications/commit/4d12e63038f14c961498002ddf808db4e5e2dbdf for the `authorizedDatabases` option.

That said, this probably doesn't matter too much.

Reply:
it looks like the `authorizedDatabases` option was added in MongoDB 4.0.5 (and 4.0 was originally numbered 3.8 iirc): https://www.mongodb.com/docs/manual/reference/command/listDatabases/#command-fields 

so perhaps that is why it was chosen (though, that same commit also notes that MongoDB 4.0.5 added the option, not 3.8).

in any case I agree this seems inconsequential.

Reply:
Reverted to 3.6 in https://github.com/mongodb/specifications/pull/1311/commits/4a476a635d706ca757c47b5d806fe2bca6a79e23.